routing-server/scratch.http

```diff
@@ -0,0 +1,21 @@
+### this only works thanks to the registry.addViewController("/").setViewName("forward:/index.html");
```

Comment:
that's what I meant, i'd like to get rid of - adding thymeleaf just for seems is overkill